### PR TITLE
update screens and add groups

### PR DESCRIPTION
**Pull request by dereckquock**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+node/
```
Comment (Phillip3) [inline .eslintignore:1]: What does this file do?

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,258 @@
+{
```
Comment (Phillip3) [inline .eslintrc:1]: What does this file do?

File: .flowconfig [hunk 3]
```diff
@@ -0,0 +1,25 @@
+[ignore]
```
Comment (Phillip3) [inline .flowconfig:1]: What does this file do?

File: .watchmanconfig [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (Phillip3) [inline .watchmanconfig]: What about this empty thing? What does this do?

File: babel.config.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+module.exports = function(api) {
```
Comment (Phillip3) [inline babel.config.js:1]: What is babel? Why do we need it?

File: .eslintrc (hunk 2 above)
Comment (dereckquock) [inline .eslintrc:1]: https://eslint.org/

there are editor integrations, if you use vscode or atom or sublime or whatevs

File: App.js [hunk 6]
```diff
@@ -1,18 +1,58 @@
...
+      <View style={styles.container}>
+        {Platform.OS === 'ios' && <StatusBar barStyle="default" />}
+        <AppNavigator />
+      </View>
+    );
+  }
+
+  _loadResourcesAsync = async () => {
+    return Promise.all([
+      Asset.loadAsync([require('./assets/images/echo_logo.png')]),
```
Comment (dereckquock) [inline App.js:31]: async load the echo logo image, so we have have it before render

File: .flowconfig (hunk 3 above)
Comment (dereckquock) [inline .flowconfig:1]: static typing! https://flow.org/

File: prettier.config.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
```
Comment (dereckquock) [inline prettier.config.js:1]: automatically format code so you dont have to worry about it https://prettier.io/

File: .eslintrc (hunk 2 above)
Comment (Phillip3) [reply .eslintrc:1]: Oh nice! cool.

File: App.js (hunk 6 above)
Comment (Phillip3) [reply App.js:31]: Nice! Although we eventually need to make sure we know what happens if there's no (or very slow) internet.

File: .flowconfig (hunk 3 above)
Comment (Phillip3) [reply .flowconfig:1]: Lol dude you've got plug-ins for days.

File: App.js (hunk 6 above)
Comment (dereckquock) [reply App.js:31]: yeah, for sure

File: .flowconfig (hunk 3 above)
Comment (dereckquock) [reply .flowconfig:1]: it's good stuff dont worry

File: .eslintignore (hunk 1 above)
Comment (dereckquock) [reply .eslintignore:1]: ignores these things when linting

File: .eslintrc (hunk 2 above)
Comment (dereckquock) [reply .eslintrc:1]: config for eslint

File: .flowconfig (hunk 3 above)
Comment (dereckquock) [reply .flowconfig:1]: config for flow typechecking

File: .watchmanconfig (hunk 4 above)
Comment (dereckquock) [reply .watchmanconfig]: actually, i think we could delete this 🤔

File: babel.config.js (hunk 5 above)
Comment (dereckquock) [reply babel.config.js:1]: https://babeljs.io/

transpiles code

File: App.js (hunk 6 above)
Comment (Phillip3) [reply App.js:31]: I stuck it in the backlog--a problem for later.

File: babel.config.js (hunk 5 above)
Comment (Phillip3) [reply babel.config.js:1]: @dereckquock do we need that with native? There's no browser compatibility to worry about.

File: .flowconfig (hunk 3 above)
Comment (Phillip3) [reply .flowconfig:1]: Lol I was more of impressed.

File: babel.config.js (hunk 5 above)
Comment (dereckquock) [reply babel.config.js:1]: https://docs.expo.io/versions/v26.0.0/guides/using-modern-javascript/#polyfills-and-babel-transforms

most of the stuff is included 👍 
but there's other stuff like `optional-chaining`

Comment (Phillip3) [reply babel.config.js:1]: meh ok. I don't have many feelings either way.

**Review by Phillip3 — DISMISSED** [review]
🔥 🔥 🔥

File: babel.config.js (hunk 5 above)
Comment (dereckquock) [reply babel.config.js:1]: it doesn't add any bloat or anything, it just makes it so that we can safely use new JS stuffs

File: .flowconfig (hunk 3 above)
Comment (dereckquock) [reply .flowconfig:1]: haha 😅

**Comment by Phillip3** [comment]
@dereckquock I'll let you merge when you're ready.

File: flow-typed/metro-bundler.js [hunk 8]
```diff
@@ -0,0 +1,9 @@
+/* @flow */
```
Comment (dereckquock) [inline flow-typed/metro-bundler.js:1]: added some of these `flow-typed` type definitions

**Review by Phillip3 — APPROVED** [review]
Approved! 🍾

**Pull request merged**